src/wp-includes/media.php

```diff
@@ -2029,6 +2029,22 @@ function wp_sizes_attribute_includes_valid_auto( string $sizes_attr ): bool {
 	return 'auto' === strtolower( trim( $first_size, " \t\f\r\n" ) );
 }
 
+/**
+ * Prints a CSS rule to fix potential visual issues with images using `sizes=auto`.
+ *
+ * This rule overrides the similar rule in the default user agent stylesheet, to avoid images that use e.g.
+ * `width: auto` or `width: fit-content` to appear smaller.
+ *
+ * @since 6.7.1
+ * @see https://html.spec.whatwg.org/multipage/rendering.html#img-contain-size
+ * @see https://core.trac.wordpress.org/ticket/62413
+ */
+function wp_print_auto_sizes_contain_css_fix() {
```

Comment:
Since we have PHP 7.4 as min version, what about adding return type here?

```suggestion
function wp_print_auto_sizes_contain_css_fix(): void {
```

Reply:
If this was for an actual returned value, I wouldn't be opposed, but given this is purely for `void`, I don't think this is something we want to "randomly" start doing in a ticket like this.

Reply:
If I see this correctly you’re adding a completely new function. 
I’m onboard with you, if this was for a changed function, but I don’t follow your thought since this is a newly introduced one. 

Just to give a second thought.

Reply:
My point is that adding `void` return types would be a new pattern that's not a thing in WordPress Core, so whether this is something that we should do in Core would need to be discussed separately. If I do it here, but then nowhere else it's done, it creates inconsistency for little value.

Reply:
That makes a lot of sense, thanks for the explanation.